src/tox/interpreters/unix.py

```diff
@@ -8,12 +10,15 @@
 
 @tox.hookimpl
 def tox_get_python_executable(envconfig):
+    """
+    Return path to specified interpreter.
+    If not available, exit(1).
+    """
     spec, path = base_discover(envconfig)
-    if path is not None:
+
+    if path is None:
+        print('envname {} not found'.format(envconfig.envname))
```

Comment:
Please don't use print, the reporter module exists instead.

Reply:
Changed to reporter.error()